modules/serverless-logic/pages/integrations/camel-routes-integration.adoc

```diff
@@ -0,0 +1,144 @@
+= Integrating with Camel routes
```

Comment:
I need to confirm if we will productize this integration.

Reply:
Well, the build of DP4 is already created, so if we didn't productize it in DP4, we can only do it in DP5 I would say.

Reply:
It's already approved, we can go ahead with it. Thanks @MarianMacik!

Reply:
@MarianMacik @radtriste the camel integration is on 1.33 only and DP4 is targeting 1.32: https://mvnrepository.com/artifact/org.kie.kogito/kogito-addons-quarkus-camel

There's something wrong in the docs branch.

Reply:
@ricardozanini that is what I had in mind, DP4 is targeting 1.32 and camel routes are only in 1.33.